Model/src/main/java/org/gusdb/wdk/model/user/dataset/event/repo/UserDatasetEventRepo.java

```diff
@@ -0,0 +1,440 @@
+package org.gusdb.wdk.model.user.dataset.event.repo;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Types;
+import java.time.LocalDateTime;
+import java.util.*;
+import javax.sql.DataSource;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gusdb.fgputil.db.runner.SQLRunner;
+import org.gusdb.wdk.model.WdkRuntimeException;
+import org.gusdb.wdk.model.user.dataset.UserDatasetType;
+import org.gusdb.wdk.model.user.dataset.event.model.UserDatasetEventStatus;
+import org.gusdb.wdk.model.user.dataset.event.model.EventRow;
+
+/**
+ * Repository for DB actions related to user dataset events.
+ * <p>
+ * UserDatasetEvent Table:
+ * <table>
+ *   <tr>
+ *     <th>Column</th>
+ *     <th>Type</th>
+ *     <th>Notes</th>
+ *   </tr>
+ *   <tr>
+ *     <td><code>EVENT_ID</code></td>
+ *     <td><code>long</code></td>
+ *     <td><code>iRODS generated event ID.</code></td>
+ *   </tr>
+ *   <tr>
+ *     <td><code>USER_DATASET_ID</code></td>
+ *     <td><code>long</code></td>
+ *     <td>ID for the user dataset this event relates to.</td>
+ *   </tr>
+ *   <tr>
+ *     <td><code>HANDLED_TIME</code></td>
+ *     <td>{@link java.time.LocalDateTime}</td>
+ *     <td>Timestamp for when the event was first handled.</td>
+ *   </tr>
+ *   <tr>
+ *     <td><code>STATUS</code></td>
+ *     <td>{@link UserDatasetEventStatus}</td>
+ *     <td>Last updated status for the event.</td>
+ *   </tr>
+ *   <tr>
+ *     <td><code>TYPE_NAME</code></td>
+ *     <td>{@link String}</td>
+ *     <td>Name of the user dataset type.</td>
+ *   </tr>
+ *   <tr>
+ *     <td><code>TYPE_VERSION</code></td>
+ *     <td>{@link String}</td>
+ *     <td>Version of the user dataset type.</td>
+ *   </tr>
+ * </table>
+ */
+public class UserDatasetEventRepo
```

Comment:
i don't understand, or am not familiar with, use of "Repo" in this context.  Maybe "Utils" is a way I'd think of it

Reply:
We've briefly talked about this for a user comments commit.  "Repo" is a name for a collection of DB interactions that was common at my previous jobs.  I don't know that "Utils" is a good fit as this class is exclusively DB calls rather than general utilities